_tips/writing.ja.md

```diff
@@ -0,0 +1,478 @@
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:".
+
+title: "ウェブ・アクセシビリティを考慮した記述法 – 始めるにあたってのヒント"
+title_html: "ウェブ・アクセシビリティを考慮した記述法"
+nav_title: 始めるためのヒント
+lang: ja  # Change "en" to the translated-language shortcode
+last_updated: 2024-03-29  # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+resource:
+  ref: /tips/  # Do not change this
+
+navigation:
+  previous: /tips/  # Do not change this
+  next: /tips/designing/  # Do not change this
+
+translators: # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
+- name: "Jay Kishigami"   # Replace Jan Doe with translator name
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple translators
+# contributors:
+# - name: "Jan Doe"   # Replace Jan Doe with contributor name, or delete this line if none
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple contributors
+
+github:
+  repository: w3c/wai-quick-start
+  path: '_tips/writing.ja.md'  # Add the language shortcode to the middle of the filename, for example: content/index.fr.md
+
+permalink: /tips/writing/ja   # Add the language shortcode to the end, with no slash at the end. For example /path/to/file/fr
+ref: /tips/writing/   # Do not change this
+
+ext_css: tips.css
+title_icon: /tips/img/icons.svg#writing
+
+acknowledgements: /tips/acknowledgements/
+
+# In the footer below:
+# Do not change the dates
+# Translate the other words, including "Date:" and "Editors:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+footer: >
+  <p><strong>月日:</strong> Updated 5 August 2022. First published September 2015.</p>
+  <p><strong>編集者:</strong> <a href="https://www.w3.org/People/kevin">Kevin White</a>, <a href="https://www.w3.org/People/shadi">Shadi Abou-Zahra</a>, and <a href="https://www.w3.org/People/Shawn">Shawn Lawton Henry</a>ACKNOWLEDGEMENTS</p>
+  <p><a href="https://www.w3.org/WAI/EO/">教育と援助のためのWG (EOWG)</a>によって開発され. <a href="https://www.w3.org/WAI/DEV/">WAI-DEV プロジェクト</a>からのサポートを受けるとともに, ヨーロッパ委員会の <abbr title="Information Society Technologies">IST</abbr> プログラムの援助を受けた.</p>
+
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="概要" class="full" %}
+{:/}
+
+このページでは､障害者にとってアクセシブルなWebコンテンツを書き始めるための考慮すべき基本的な事項を紹介します｡これらのヒントは､Webコンテンツ・アクセシビリティ・ガイドライン(WCAG)の要件を満たすためのグッドプラクティスです｡関連するWCAG要件､「理解するための」ドキュメントでの詳細な背景､チュートリアルからのガイダンス､ユーザーストーリーなどへのリンクをたどってください｡
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="目次" class="full" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## 分かりやすくユニークなページタイトルを
```

Comment:
@jaykishigami The original text is "Provide informative, unique page titles". Can you confirm to me the translation does indeed convey the idea that a page title should be "informative"?

My automatic tool translate it into "clear and unique" rather than "informative and unique" so I just want to check it with you.

Reply:
## 情報量が多くてユニークなページタイトルを